server/controllers/UserController.js

```diff
@@ -0,0 +1,220 @@
+import jwt from 'jsonwebtoken';
+import model from '../models';
+
+const User = model.User;
+const Documents = model.Document;
+const secret = 'secret';
+
+
+const UserController = {
+  login(request, response) {
+    if (request.body.loginId === '' || request.body.password === '') {
+      return response.status(400).json({
+        status: 'Bad Request',
+        message: 'Fields cannot be empty'
+      });
+    }
+    return User
+      .findOne({
+        where: {
+          $or: [{
+            username: request.body.loginId
+          }, {
+            email: request.body.loginId
+          }]
+        }
+      })
+      .then((user) => {
+        if (!user) {
+          return response.status(404).send({
+            status: 404,
+            message: 'User does not exist'
```

Comment:
for security purpose, I don't think it's ideal to let the user know if the username if right or wrong. Simple invalid login details could suffice.